pkg/oc/cli/admin/ipfailover/validation.go

```diff
@@ -65,23 +71,39 @@ func ValidateIPAddressRange(iprange string) error {
 	return nil
 }
 
-// ValidateVirtualIPs validates virtual IP range/addresses.
-func ValidateVirtualIPs(vips string) error {
+// validateVirtualIPs validates virtual IP range/addresses.
+func validateVirtualIPs(vips string) error {
 	virtualIPs := strings.TrimSpace(vips)
 	if len(virtualIPs) < 1 {
 		return nil
 	}
 
 	for _, ip := range strings.Split(virtualIPs, ",") {
-		if err := ValidateIPAddressRange(ip); err != nil {
+		if err := validateIPAddressRange(ip); err != nil {
 			return err
 		}
 	}
 
 	return nil
 }
 
-// ValidateCmdOptions validates command line operations.
-func ValidateCmdOptions(options *IPFailoverConfigCmdOptions) error {
-	return ValidateVirtualIPs(options.VirtualIPs)
+func validateServiceAccount(client securityv1typedclient.SecurityV1Interface, serviceAccount string) error {
+	sccList, err := client.SecurityContextConstraints().List(metav1.ListOptions{})
+	if err != nil {
+		if !errors.IsUnauthorized(err) {
```

Comment:
```go
if err != nil && errors.IsUnathorized(err) {
...
``` 
???

Reply:
Heh, will fix this up. I pretty much just copy + pasted it exactly as it was before this PR (it used to live in the ipfailover.go file)